cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureStorage.java

```diff
@@ -153,7 +153,7 @@ private String buildStorageName(ArmAttachedStorageOption armAttachedStorageOptio
         name = name.length() > MAX_LENGTH_OF_NAME_SLICE ? name.substring(0, MAX_LENGTH_OF_NAME_SLICE) : name;
         try {
             MessageDigest messageDigest = MessageDigest.getInstance("MD5");
-            String storageAccountId = acv.getCredentialCrn() + "#" + cloudContext.getId() + '#' + cloudContext.getAccountId();
+            String storageAccountId = acv.getCredentialCrn() + "#" + cloudContext.getId() + '#' + cloudContext.getTenantId();
```

Comment:
Is this logic relevant for `freeipa` service as well? If so, it will likely not populate the new property, so `getTenantId()` will remain `null` (while `getAccountId()` will likely contain the account UUID as before).

Reply:
yes, you are right, it needs correction